### ISSUE TITLE
datalad get  barfes

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#227]
running version of #199 
```
$> datalad get hyperalignment_tutorial_data.hdf5.gz
2015-09-21 11:26:49,025 [ERROR  ] type object 'AnnexRepo' has no attribute 'startswith' (AttributeError) (main.py:206)
```
which shows how much we need those all kinds of tests for our API commands even if they are basic
```
$> datalad  --dbg get hyperalignment_tutorial_data.hdf5.gz   
Traceback (most recent call last):
  File "/home/yoh/proj/datalad/datalad-debian/venv-tests/bin/datalad", line 9, in <module>
    load_entry_point('datalad==0.0.3.dev0', 'console_scripts', 'datalad')()
  File "/home/yoh/proj/datalad/datalad-debian/datalad/cmdline/main.py", line 199, in main
    cmdlineargs.func(cmdlineargs)
  File "/home/yoh/proj/datalad/datalad-debian/datalad/interface/base.py", line 149, in call_from_parser
    return self(**kwargs)
  File "/home/yoh/proj/datalad/datalad-debian/datalad/interface/get.py", line 41, in __call__
    handle = get_repo_instance(AnnexRepo)
  File "/home/yoh/proj/datalad/datalad-debian/datalad/cmdline/helpers.py", line 161, in get_repo_instance
    dir_ = abspath(expandvars(expanduser(path)))
  File "/home/yoh/proj/datalad/datalad-debian/venv-tests/lib/python2.7/posixpath.py", line 254, in expanduser
    if not path.startswith('~'):
AttributeError: type object 'AnnexRepo' has no attribute 'startswith'
```

**Comment by yarikoptic** [comment]
assigned to myself for now -- want to try to come up with a unittest for this one

**Comment by yarikoptic** [comment]
This one was fixed in #231
